public/app/plugins/datasource/prometheus/querybuilder/shared/QueryBuilderHints.tsx

```diff
@@ -45,6 +46,11 @@ export const QueryBuilderHints = <T extends PromLokiVisualQuery>({
               <Tooltip content={`${hint.label} ${hint.fix?.label}`} key={hint.type}>
                 <Button
                   onClick={() => {
+                    reportInteraction('grafana_explore_query_builder_hints_clicked', {
```

Comment:
```suggestion
                    reportInteraction('grafana_query_builder_hints_clicked', {
```

Sorry that I haven't noticed this before, but query builders are present in both - explore and dashboard and alerting,... . So here we should make it more generic and remove "explore" from the name.